lib/puppet/provider/package/yum.rb

```diff
@@ -155,7 +155,18 @@ def install
     self.debug "Ensuring => #{should}"
     operation = :install
 
+    current_package = self.query
```

Comment:
This was already in from 3180b9d9b2c844dade1d361326600f7001ec66dd, whats a better way?